Compilation error

## Problem
There is a compilation error on devel:
```
privmx-endpoint/utils/test/VersionNumber.cpp:18:10: fatal error: gmpxx.h: Nie ma takiego pliku ani katalogu
   18 | #include <gmpxx.h>
      |          ^~~~~~~~~
```

Btw. Including the gmpxx.h is not needed for that test code.

--- a/utils/test/VersionNumber.cpp
+++ b/utils/test/VersionNumber.cpp
@@ -15,7 +15,6 @@ limitations under the License.
 
 #include <privmx/utils/VersionNumber.hpp>
 #include "privmx/utils/PrivmxExtExceptions.hpp"
-#include <gmpxx.h>
 
 using namespace std;
 

